index.js

```diff
@@ -154,46 +165,45 @@ const recastNewUsers = async () => {
     if (USERNAMES_TO_SKIP.includes(user.username)) {
       continue
     }
-    if (recastCounts[user.address] >= MAX_RECAST_PER_USER) {
+    if (recastCounts[user.fid] >= MAX_RECAST_PER_USER) {
       console.log(
         `Skipping ${user.username} because casted more than ${MAX_RECAST_PER_USER} times already.`
       )
       continue // Skip because exceeded max recasts
     }
-    const casts = await getCasts(user.address)
-    let recastCount = recastCounts[user.address]
-    for (const cast of casts.result.casts.reverse()) {
+    const casts = await getCasts(NCBOT_FID)
+    let recastCount = recastCounts[user.fid]
+    for (const cast of casts.result.casts) {
```

Comment:
has the ordering of casts changed (re: removing `.reverse()`)? ideally we recast from earliest to latest.

Reply:
will fix this, wasn't aware of that